hardware/unicorn/unihat.py

```diff
@@ -6,6 +6,11 @@
 import os
 import unicornhat as UH
 
+try:
+    raw_input          # Python 2
+except NameError:
+    raw_input = input  # Python 3
+
 if sys.version_info >= (3,0):
```

Comment:
hmm - guess this can now be removed ?

Reply:
https://pypi.org/project/unicornhat is Python 3 compatible but I think that https://github.com/node-red/node-red-nodes/blob/master/hardware/unicorn/scripts/checklib.js#L4 still stands in our way.